### ISSUE TITLE
Last doesn't give the last result.

### ISSUE THREAD

**Issue by BartWillems** [gobuffalo/pop#45]
The ".Last()" function uses the following statement:

``` sql
SELECT table.column, ...
FROM table AS alias 
ORDER BY id desc LIMIT 1 
```

Because fizz uses UUIDs as ids by default, ordering by id wil yield a random result. It should order by created_at

**Comment by markbates** [comment]
A PR for this would be great! Good find.

**Comment by stanislas-m** [comment]
Fixed with #47.
